crosschain-resolver/contracts/IMetadataResolver.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+interface IMetadataResolver {
+    /**
+     * @notice Get metadata about the CCIP Resolver
+     * @dev This function provides metadata about the CCIP Resolver, including its name, coin type, GraphQL URL, storage type, and encoded information.
+     * @param name The domain name in format (dnsEncoded)
+     * @return name The name of the resolver ("CCIP RESOLVER")
+     * @return coinType Resolvers coin type (60 for Ethereum)
+     * @return graphqlUrl The GraphQL URL used by the resolver
+     * @return storageType Storage Type (0 for EVM)
```

Comment:
This needs clearer documentation on available values.

Reply:
Cuurently the ENS IP defines just `* @return storageType 0 = EVM, 1 = Non blockchain, 2 = Starknet` Do we need more definitions or should I just put the link to the ENSIP so we can maintain the up to date list in the doc?